### ISSUE TITLE
Apply default BOOT_JVM_OPTIONS (will fix heap space issues for some users)

### ISSUE THREAD

**Issue by gregsabo** [alda-lang/alda#42]
Here's what I get when I run alda on OS X 10.10.5:
```
~|⇒ alda repl
2015-Sep-06 13:34:32 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.attribute...
2015-Sep-06 13:34:32 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.offset...
2015-Sep-06 13:34:32 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.global-attribute...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.marker...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.event...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.pitch...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.duration...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.model.instrument...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.attributes...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.events.note...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.events.rest...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.events.chord...
2015-Sep-06 13:34:33 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.events.voice...
2015-Sep-06 13:34:34 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.instruments.midi...
2015-Sep-06 13:34:35 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.score.part...
2015-Sep-06 13:34:35 -0700 UnknownHost DEBUG [alda.lisp] - Loading alda.lisp.score...

 █████╗ ██╗     ██████╗  █████╗
██╔══██╗██║     ██╔══██╗██╔══██╗
███████║██║     ██║  ██║███████║
██╔══██║██║     ██║  ██║██╔══██║
██║  ██║███████╗██████╔╝██║  ██║
╚═╝  ╚═╝╚══════╝╚═════╝ ╚═╝  ╚═╝

            v0.4.4
         repl session

Loading MIDI synth... 2015-Sep-06 13:34:36 -0700 UnknownHost DEBUG [alda.sound.midi] - Loading MIDI synth...
2015-Sep-06 13:34:36 -0700 UnknownHost DEBUG [alda.sound.midi] - Loading MIDI soundfont...
clojure.lang.ExceptionInfo: Java heap space
    data: {:file
           "/var/folders/_g/vjftnz6s2q36lcsmxzw3ypl80000gn/T/boot.user4692146052994996114.clj",
           :line 11}
java.lang.OutOfMemoryError: Java heap space
     com.sun.media.sound.SF2Soundbank.readSdtaChunk                 SF2Soundbank.java: 177
     com.sun.media.sound.SF2Soundbank.readSoundbank                 SF2Soundbank.java: 132
            com.sun.media.sound.SF2Soundbank.<init>                 SF2Soundbank.java: 113
com.sun.media.sound.SF2SoundbankReader.getSoundbank           SF2SoundbankReader.java:  58
           javax.sound.midi.MidiSystem.getSoundbank                   MidiSystem.java: 530
                                                ...
               midi.soundfont/load-all-instruments!                     soundfont.clj:  13
                   alda.sound.midi/open-midi-synth!                          midi.clj:  65
                             alda.sound/eval7701/fn                         sound.clj:  23
                                                ...
                                 alda.sound/set-up!                         sound.clj:  33
                                                ...
                              alda.repl/start-repl!                          repl.clj:  91
                               alda.cli/eval8100/fn                           cli.clj:  66
                                                ...
                                  alda.cli/delegate                           cli.clj:  91
                                     alda.cli/-main                           cli.clj: 105
                                                ...
                                 clojure.core/apply                          core.clj: 624
                                    boot.user/-main  boot.user4692146052994996114.clj:   9
                                                ...
```

**Comment by daveyarwood** [comment]
This is a JVM settings issue -- you can fix it by following the instructions here: https://github.com/boot-clj/boot/wiki/JVM-Options

Here is my `$BOOT_JVM_OPTIONS`, for reference:

```
$ echo $BOOT_JVM_OPTIONS
-client -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xmx2g -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -Xverify:none
```

It might be good to find a way for us to apply this setting automatically, though, for the sake of those of us who aren't used to mucking around with environment variables :)

**Comment by daveyarwood** [comment]
This issue will be solved in the near future when Alda will not require Boot to run.

Should be fixed with the upcoming 1.0.0-rc1 release candidate.
